roles/signalilo/tasks/main.yml

```diff
@@ -1,3 +1,17 @@
+- name: Include os specific vars
```

Comment:
- this task should be in `source.yml` as it is only needed by source installation method
- task should be delegated to `localhost` as build is not intended to be on target host
- by using `become: true` (which this task requires), playbook executor must have local privileged access. This is not always possible. Suggestion: instead test for presence of `make` and `go`, fail if not available. Installing golang in user space is simple (https://go.dev/doc/install) and allows choice to do it either as user or system-wide.